packages/experimental/Menu/src/MenuPopover/useMenuPopover.ts

```diff
@@ -1,10 +1,17 @@
+import React from 'react';
+import { DismissBehaviors } from '@fluentui-react-native/callout';
 import { useMenuContext } from '../context/menuContext';
 import { MenuPopoverProps, MenuPopoverState } from './MenuPopover.types';
 
 export const useMenuPopover = (_props: MenuPopoverProps): MenuPopoverState => {
   const context = useMenuContext();
+  const setOpen = context.setOpen;
 
   const triggerRef = context.triggerRef;
+  const onDismiss = React.useCallback(() => setOpen(undefined, false /* isOpen */), [setOpen]);
+  const dismissBehaviors = context.isControlled
+    ? (['preventDismissOnKeyDown', 'preventDismissOnClickOutside'] as DismissBehaviors[])
+    : undefined;
```

Comment:
Feels like a good followup in Popover V1 to simplify this bit on the native side..